plugins/scaffolder/src/plugin.tsx

```diff
@@ -73,6 +73,8 @@ import {
   MyGroupsPicker,
   MyGroupsPickerSchema,
 } from './components/fields/MyGroupsPicker/MyGroupsPicker';
+import { RepoBranchPicker } from './components/fields/RepoBranchPicker/RepoBranchPicker';
+import { RepoBranchPickerSchema } from './components/fields/RepoBranchPicker/schema';
```

Comment:
This will only work if we also export `RepoBranchPickerSchema` from `index.ts`. Do we want to do that? Are there any rules to how we treat barrel files? (the current implementation is just how it is in the `RepoUrlPicker`)

Reply:
It's alright to keep as is if we want to avoid exporting these from `RepoBranchPicker`